app/models/organization.rb

```diff
@@ -46,6 +46,7 @@ class Organization < ApplicationRecord
   enum document_numbering: DOCUMENT_NUMBERINGS
 
   before_create :generate_api_key
+  before_update -> { self.document_number_prefix = document_number_prefix.upcase }
```

Comment:
Could you explain the reason behind this? I'm not sure to understand why it should be before each update.

Reply:
We don't allow small cases and this field is editable:) I guess other option is moving it to service level...